docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -13,13 +13,13 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
 Excel add-ins run across multiple versions of Office, including Office 2016 or later for Windows, Office for iPad, Office for Mac, and Office Online. The following table lists the Excel requirement sets, the Office host applications that support each requirement set, and the build versions or number for those applications.
 
 > [!NOTE]
-> Any API that is marked as **Beta** is not ready for end-user production. We make them available for developers to try them out in test and development environments. They are not meant to be used against production/business critical documents.
-> 
-> For the requirement sets that are marked as **Beta**, use the specified (or later) version of the Office software and use the Beta library on the CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js. Entries not marked as **Beta** are generally available and you can use Production library on the CDN: https://appsforoffice.microsoft.com/lib/1/hosted/office.js.
+> To use APIs in any of the numbered requirement sets, you should reference the **production** library on the CDN: https://appsforoffice.microsoft.com/lib/1/hosted/office.js.
+>
+> For information about using preview APIs, see the [Excel JavaScript preview API set](#excel-javascript-preview-api-set) section within this article.
```

Comment:
If you change the section heading as I've recommended below, you'll need to change the hyperlink text here to match the new section heading -> [Excel JavaScript preview APIs](excel-javascript-preview-apis)